pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -257,6 +258,18 @@ func (r *ReconcileOperConfig) Reconcile(ctx context.Context, request reconcile.R
 		return reconcile.Result{}, nil
 	}
 
+	// Only allows live migration on SD managed clusters
+	inManagedCluster, err := util.IsRunningInManagedCluster(ctx, r.client)
+	if err != nil {
+		log.Println(err)
+		return reconcile.Result{}, err
+	}
+	if !inManagedCluster && operConfig.Spec.Migration != nil && operConfig.Spec.Migration.Mode == operv1.LiveNetworkMigrationMode {
+		r.status.SetDegraded(statusmanager.OperatorConfig, "MergeClusterConfig",
```

Comment:
so ultimately anyone can set the API value and we are going to make the operator force itself to become degraded? I hope we are doing release notes for this, seems like we support the API officially but one can't use it. Also is this API still in TechPreviewNoUpgrade and this is the requirement from API team for them to allow this in normal clusters (sorry I am missing context on this one)?

Reply:
ack thanks @martinkennelly that deff sets the context for me. Again not a fan of this change but if this is what we have to do then so be it.